client/sections.js

```diff
@@ -133,8 +133,8 @@ sections = [
 	{
 		name: 'theme',
 		paths: [ '/theme' ],
-		module: 'my-sites/themes',
-		enableLoggedOut: config.isEnabled( 'manage/themes/logged-out' ),
+		module: 'my-sites/theme',
+		enableLoggedOut: true,
```

Comment:
Yeah. I think `manage/themes/logged-out` was mainly to contain related changes for M3 for the `/design` route, so I didn't think it applied to `/theme`. For `/theme`, I think it's fine to not have an extra flag for logged-out, since we've been designing it with logged-out and even SSR in mind from the beginning.